validator/tests/test_journal/tests.py

```diff
@@ -864,12 +865,20 @@ def chain_updated(head, committed_batches=None,
         # Patch the threadpool
         self.executor = SynchronousExecutor()
         self.block_validator._thread_pool = self.executor
+
+        # Set genesis after validating
+        genesis = self.block_tree_manager.generate_genesis_block()
+        self.block_validator.validate_block(genesis)
+        self.block_tree_manager.set_chain_head(genesis)
+        self.chain_ctrl._set_chain_head_from_block_store()
+
         init_root = self.chain_ctrl.chain_head
         self.assert_is_chain_head(init_root)
 
         # create a chain of length 5 extending the root
-        _, head = self.generate_chain(init_root, 5)
-        self.receive_and_process_blocks(head)
+        chain, head = self.generate_chain(
+            init_root, 5, {"add_to_store": False, "add_to_cache": False})
```

Comment:
`generate_chain` should probably take `{"add_to_store": False, "add_to_cache": False}` as a default argument; passing it in to every call adds a lot of noise to the tests.